toc_config.py

```diff
@@ -163,9 +164,7 @@ def parse_chapter(docname, doc, parent):
         category = u'chapter'
         for name,hidden,child in traverse_tocs(app, doc):
             meta = first_meta(child)
-            status = u'hidden' if 'hidden' in meta else (
-                u'unlisted' if hidden else u'ready'
-            )
+            status = u'status'
```

Comment:
This doesn't seem right. I have now idea where the `hidden` variable comes from and when there is `hidden` in meta. I recommend you look into this more and explain to me :)

Reply:
You are right, this is wrong. This defines chapter status, not exercise status. The task is to make submit exercises' statuses manually determinable, and chapter status should remain untouched.